test/benchmarks/quantum_info.py

```diff
@@ -85,3 +88,95 @@ def time_compose(self, nqubits_length):
         cxdihedral = CNOTDihedral(num_qubits=nqubits)
         for i in range(length):
             cxdihedral.compose(self.random_cnotdihedral[i])
+
+
+class PauliBench:
+    params = [100, 200, 300, 400, 500]
```

Comment:
Can you also set a `param_names` attribute here so the graphs render this as num_qubits:

```suggestion
    params = [100, 200, 300, 400, 500]
    param_names = ["num_qubits"]
```